src/EditorFeatures/Core/Implementation/Suggestions/FixAllGetFixesService.cs

```diff
@@ -133,7 +105,7 @@ private CodeAction GetFixAllCodeAction(FixAllProvider fixAllProvider, FixAllCont
                 newSolution = PreviewChanges(
```

Comment:
PreviewChanges seems to work fine when i smoke test making a project/solution change.  THere does not appear to be a problem with the two dialogs.